get-thin-edge_io.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Someone that is not me must decide whether this is okay.

Reply:
For a quick install script `sh` would make more sense than `bash`.

What is specific to `bash` in this script? I noticed that you properly replaced `if apt -v &>/dev/null` by `apt -v >/dev/null 2>&1`. So `sh` should work.

Reply:
> What is specific to bash in this script?

The escape sequences that are printed with `echo -e`. The `-e` flag is not specified by POSIX.

Reply:
In that case, I prefer to avoid the `echo -e` and use `sh`.